6-Typescript/server/tests/it/create-todo.test.ts

```diff
@@ -1,31 +1,34 @@
-import { createTodo, createTodoWithMissingChecked } from './../env/random';
-import { TestKit } from '../env/TestKit';
-import { ITodo } from '../../interfaces/ITodo';
+import { createTodo, createTodoWithMissingChecked } from "../env/mockData";
+import { TestKit } from "../env/TestKit";
+import { ITodo } from "../../interfaces/interfaces";
 describe("POST new todo /api/todos ", () => {
-    const testKit = new TestKit()
+    const testKit = new TestKit();
     testKit.beforeAndAfter();
 
     describe("given new todo to create ", () => {
```

Comment:
why do you need a dedicated `describe` for one test?
You should use the `describe` to make few related tests and maybe even use `beforeEach` for them (that's where the `given` in the description comes from)